### PR TITLE
[Paddle Backend] Add spin energy example(revert code format)

### PR DESCRIPTION
**Pull request by HydrogenSulfate**
code formated from #3028

### CODE REVIEW

File: deepmd/loss/ener.py [hunk 1]
```diff
@@ -365,151 +363,202 @@ def build(self, learning_rate, natoms, model_dict, label_dict, suffix):
...
-            name="l2_atom_ener_" + suffix,
+        atom_ener_pred_reshape = paddle.reshape(atom_ener_pred, [-1])
+        atom_ener_label_reshape = paddle.reshape(atom_ener_label, [-1])
+        l2_atom_ener_loss = (
+            paddle.mean(
+                paddle.square(atom_ener_pred_reshape - atom_ener_label_reshape),
+                name="l2_atom_ener_" + suffix,
+            )
+            if self.has_ae
+            else 0.9
```
Comment (njzjz) [inline deepmd/loss/ener.py:454]: I guess it is a typo..
```suggestion
            else 0.0
```

Comment (HydrogenSulfate) [reply deepmd/loss/ener.py]: Fixed, thanks!

**Review by njzjz — APPROVED** [review]
